src/App.jsx

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+import backgroundImg from "./shared/assets/backgroundImg.jpg";
+
+import MenuComponent from "./components/MenuComponent";
+
+const ContentWrapper = styled.div`
+  display: grid;
+  grid-template-rows: 93vh 90px;
+`;
+
+const Cover = styled.img`
```

Comment:
👍🏻

Reply:
naamgeving zou hier nog duidelijker kunnen: `CoverBackgroundImage` of iets dergelijks?